packaging/build_packages_wf.json

```diff
@@ -108,14 +112,15 @@
             {
               "initializeParams": {
                 "diskType": "pd-ssd",
-                "sourceImage": "projects/rhel-cloud/global/images/family/rhel-8"
+                "sourceImage": "projects/centos-cloud/global/images/family/centos-8"
```

Comment:
why this switch? they're somewhat dissimilar atm

Reply:
Speed. I found building on centos was much faster, at least in the past (also cheaper)